kpatch-build/lookup.c

```diff
@@ -338,7 +341,19 @@ struct lookup_table *lookup_open(char *symtab_path, char *symvers_path,
 
 void lookup_close(struct lookup_table *table)
 {
+	int i;
+	struct object_symbol *obj_sym;
+	struct export_symbol *exp_sym;
+
+	for_each_obj_symbol(i, obj_sym, table) {
+			free(obj_sym->name);
+	}
```

Comment:
Please fix the indentation here and in the following block. We do not require the braces for single statement in the for loop.

Reply:
Have fixed your review, thanks !